client/src/modules/authorization/model/reducer.js

```diff
@@ -1,8 +1,6 @@
-import { combineReducers } from 'redux';
-
 import { AuthorizationActionTypes } from './actions';
 
-export const data = (state = null, action) => {
+const user = (state = null, action) => {
```

Comment:
be aware to not overwrite 'currentUser' from previous branch

Reply:
I changed it back to prevent this situation you wrote me about.